examples/cds-feature-attachments-oss/src/main/java/com/sap/cds/feature/attachments/oss/client/OSService.java

```diff
@@ -0,0 +1,91 @@
+package com.sap.cds.feature.attachments.oss.client;
+
+import java.io.InputStream;
+import java.time.Instant;
+
+import com.sap.cds.feature.attachments.service.model.service.AttachmentModificationResult;
+import com.sap.cds.feature.attachments.service.model.service.MarkAsDeletedInput;
+import com.sap.cds.services.Service;
+import com.sap.cds.services.ServiceException;
+
+/**
+ * The {@link OSService} is the connection to the storage for attachments.
+ */
+public interface OSService extends Service {
```

Comment:
This interface is not required, it's just a copy of https://github.com/cap-java/cds-feature-attachments/blob/main/cds-feature-attachments/src/main/java/com/sap/cds/feature/attachments/service/AttachmentService.java